core/ktx/src/main/java/de/mm20/launcher2/ktx/Context.kt

```diff
@@ -24,6 +25,19 @@ fun Context.tryStartActivity(intent: Intent, bundle: Bundle? = null): Boolean {
         startActivity(intent, bundle)
         true
     } catch (e: ActivityNotFoundException) {
+
+        if (intent.data?.path == "/storage/emulated/0/Download") {
+            val aospDocsViewDownloadsIntent = Intent()
+                .setComponent(ComponentName("com.android.documentsui", "com.android.documentsui.ViewDownloadsActivity"))
+
+            return try {
+                startActivity(aospDocsViewDownloadsIntent, bundle)
+                true
+            } catch (_: Exception) {
+                false
+            }
+        }
+
```

Comment:
This should be moved to https://github.com/MM2-0/Kvaesitso/blob/4c0b34e091e0779a164fd60eeaa0941dcb2da916/data/files/src/main/java/de/mm20/launcher2/files/providers/LocalFile.kt#L189. Less hacky that way.